### PR TITLE
FSOC 126 - If solution flag is present during push, the solution will be subscribed to

### PR DESCRIPTION
**Pull request by GDW1**
## Description

Please provide a meaningful description of what this change will do, or is for. Bonus points for including links to
related issues, other PRs, or technical references.

Note that by _not_ including a description, you are asking reviewers to do extra work to understand the context of this
change, which may lead to your PR taking much longer to review, or result in it not being reviewed at all.

## Type of Change

- [ ] Bug Fix
- [x] New Feature
- [ ] Breaking Change
- [ ] Refactor
- [ ] Documentation
- [ ] Other (please describe)

## Checklist

- [x] I have read the [contributing guidelines](/CONTRIBUTING.md)
- [x] Existing issues have been referenced (where applicable)
- [x] I have verified this change is not present in other open pull requests
- [ ] Functionality is documented
- [x] All code style checks pass
- [ ] New code contribution is covered by automated tests
- [ ] All new and existing tests pass

### CODE REVIEW

**Review by pnickolov — CHANGES_REQUESTED** [review]
comments sent separately

File: cmd/solution/upload.go [hunk 1]
```diff
@@ -229,6 +229,29 @@ func uploadSolution(cmd *cobra.Command, push bool) {
 		output.PrintCmdStatus(cmd, fmt.Sprintf("Installed %v successfully.\n", solutionDisplayText))
 	}
 
+	if cmd.Flag("subscribe").Value.String() == "true" {
```
Comment (pnickolov) [inline cmd/solution/upload.go:232]: can you use Value.Bool() ?

**Review by pnickolov — CHANGES_REQUESTED** [review]

File: cmd/solution/upload.go (hunk 1 above)
Comment (GDW1) [reply cmd/solution/upload.go:232]: image

unfortunately, Cobra does not implement a boolean value function on their Value object for flags. I could alternatively convert to boolean value manually, though I think that the above solution works better.

Comment (pnickolov) [reply cmd/solution/upload.go:232]: sorry, I meant `cmd.Flags().GetBool("subscribe")`

**Review by pnickolov — CHANGES_REQUESTED** [review]

**Review by pnickolov — APPROVED** [review]

**Pull request merged**